src/types.ts

```diff
@@ -18,6 +18,8 @@ export interface PlayerType {
   teamId: number | null;
   controllerFix: string | null;
   nametag: string | null;
+  displayName: string;
+  connectCode: string;
```

Comment:
Should these not be nullable fields too? Especially from replays created on console or in VS Mode?